edsnlp/pipes/trainable/ner_crf/ner_crf.py

```diff
@@ -506,10 +546,21 @@ def postprocess(
     ):
         spans: Dict[Doc, list[Span]] = defaultdict(list)
         contexts = [ctx for sample in inputs for ctx in sample["$contexts"]]
-        tags = results["tags"].cpu()
+        tags = results["tags"]
+        if self.compute_prob:
+            probs = results["probs"]
+
         for ctx, label, start, end in self.crf.tags_to_spans(tags).tolist():
             span = contexts[ctx][start:end]
             span.label_ = self.labels[label]
+            if self.compute_prob:
+                span_probs = probs[ctx, start:end, label, :]
+                average_entity_confidence_score = torch.mean(
+                    1 - span_probs[:, 0]
+                ).item()
+                span._.ner_prob["average_entity_confidence_score"] = (
```

Comment:
Now I'm wondering, if we output a "..._confidence_score", and not really a probability (although it's still between 0 and 1), should we instead name the feature `compute_confidence_score` instead of `compute_prob` ?